Maple2.Server.Login/PacketHandlers/CharacterManagementHandler.cs

```diff
@@ -104,6 +104,22 @@ private enum Command : byte {
         var jobCode = packet.Read<JobCode>();
         var job = (Job) ((int)jobCode * 10);
         string name = packet.ReadUnicodeString();
+        
+        if (name.Length < 2) {
+            session.Send(CharacterListPacket.CreateError(s_char_err_name));
+            return;
+        }
+        
+        if (name.Length > 12) {
```

Comment:
`Constant.CharacterNameLengthMax`

I would also just choose one of the errors to return here rather than do nothing.